portals/publisher/src/main/webapp/source/src/app/components/GlobalPolicies/Policies/GlobalSpecificComponents/Policies.tsx

```diff
@@ -0,0 +1,610 @@
+            updateGlobalOperations,
+            deleteGlobalOperation,
+            rearrangeGlobalOperations
+        }),
+        [
+            globalLevelPolicies,
+            updateGlobalOperations,
+            deleteGlobalOperation,
+            rearrangeGlobalOperations
+        ],
+    );
+
+    /**
+     * Handle Name field changes.
+     * @param {any} event changing event.
+     */
+    const handleNameChange = (event: any) => {
+        setName(event.target.value);
+    };
+    
+    /**
+     * Handle Description field changes.
+     * @param {any} event changing event.
+     */
+    const handleDescriptionChange = (event: any) => {
+        setDescription(event.target.value);
+    };
+
+    /**
+     * Loading screen if loading is true or there is no policies yet.
+     */
+    if (!policies || loading) {
+        return <Progress per={90} message='Loading Policies ...' />
+    }
+
+    return (
+        <GlobalPolicyContextProvider value={providerValue}>
+            <Box mt={3} mb={3} ml={5} mr={5}>   
+                <DndProvider backend={HTML5Backend}>
+                    {/**
+                    * Breadcrumb Navigation.
+                    */}
+                    <Grid item md={12}>
+                        <div className={classes.titleWrapper}>
+                            <Link to='/global-policies' className={classes.titleLink}>
+                                <Typography variant='h4' component='h1'>
+                                    <FormattedMessage
+                                        id='globalPolicies.heading'
+                                        defaultMessage='Global Policies'
+                                    />
+                                </Typography>
+                            </Link>
+                            <Icon>keyboard_arrow_right</Icon>
+                            <Typography variant='h4' component='h2'>
+                                {isCreateNew ? 
+                                    <FormattedMessage
+                                        id='globalPolicies.create.create.heading'
+                                        defaultMessage='Create A New Global Policy'
+                                    />
+                                    : 
+                                    <FormattedMessage
+                                        id='globalPolicies.create.edit.heading'
+                                        defaultMessage='Edit Global Policy'
+                                    />
+                                }
+                            </Typography>
+                        </div>
+                    </Grid>
+
+                    {/**
+                    * Name & Description Fields.
+                    */}
+                    <Box mb={2}>
+                        <TextField
+                            fullWidth
+                            required
+                            id='outlined-required'
+                            label='Name'
```

Comment:
It is good if we can use a formatted message for the label if possible. Let's fix other places too in this PR.

Reply:
Addressed the required changes by [fddf57f](https://github.com/wso2/apim-apps/pull/515/commits/fddf57fa7308a53094c808e9bf2d2b40ed71b644) and [0ad490c](https://github.com/wso2/apim-apps/pull/515/commits/0ad490c497b08f999babbe521feaf83d029a6698)